src/components/common/Button/index.stories.tsx

```diff
@@ -13,10 +13,8 @@ export default {
   decorators: [ThemeToggler],
   argTypes: {
     label: { control: 'text' },
-    kind: {
-      control: 'inline-radio',
-      options: ButtonVariations,
-    },
+    kind: { control: 'radio' },
```

Comment:
`type` feels more natural to me vs `kind`. But not sure if `type` is reserved as a default HTML attribute.

Reply:
^ good usecase for the `$varName` nomenclature .

Yes i want to change this name i hate it, but haven't prioritised it yet. Will do that soon

Reply:
Don't do `type`, buttons have their native type attribute

Reply:
@Velenir That's what I thought, it will collide most likely.

Reply:
fixed in https://github.com/gnosis/dex-react/pull/1637/commits/e44063a8b54827421391f5fb9ccc1d9530c8e4d2